compatibility_lib/compatibility_lib/test_get_compatibility_data.py

```diff
@@ -72,30 +72,6 @@ def mock_init():
             'compatibility_lib.get_compatibility_data.store',
             self.fake_store)
 
-    def test__generate_pairs_for_github_head(self):
```

Comment:
Can you explain why this test is removed?

Reply:
This is no longer needed, as we don't write to bigquery for gh head versions if we have the cache for commit numbers.

Reply:
Ok - the reason I asked is because I saw you moved the function `_generate_pairs_for_github_head()` to compatibility_lib/compatibility_lib/utils.py

If this function is no longer needed, you should probably remove it from that file as well.

Reply:
Done.